api/v1alpha1/cdapmaster_types.go

```diff
@@ -184,8 +184,12 @@ type CDAPExternalServiceSpec struct {
 	CDAPScalableServiceSpec `json:",inline"`
 	// ServiceType is the service type in kubernetes, default is NodePort.
 	ServiceType *string `json:"serviceType,omitempty"`
+	// LoadBalancerIP is the static IP address reserved for the service's load balancer.
+	LoadBalancerIP *string `json:"loadBalancerIP,omitempty"`
 	// ServicePort is the port number for the service.
 	ServicePort *int32 `json:"servicePort,omitempty"`
+	// ServiceAnnotations are the metadata annotations for the kubernetes service. They can be used to configure cloud provider specific behavior for LoadBalancer services.
+	ServiceAnnotations map[string]string `json:"serviceAnnotations,omitempty"`
```

Comment:
Just call it annotations, as it is already inside the service object.

Reply:
Changed. It was initially named ServiceAnnotations because a `metav1.ObjectMeta` struct is embedded inside `CDAPServiceSpec` and  calling CDAPServiceSpec.Annotations would give the annotations of the embedded object. After introducing a field named `Annotations` in this struct the annotations in the embedded struct can be accessed using `CDAPServiceSpec.ObjectMeta.Annotations`.